app/controllers/instrument_request_approval_controller.rb

```diff
@@ -0,0 +1,20 @@
+class InstrumentRequestApprovalController < ApplicationController
+  before_filter :load_instrument
+
+  def new; end
+
+  def create
+    TrainingRequest.create!(user: current_user, product: @instrument)
```

Comment:
I'm assuming this should never be invalid, hence why we would want to see exceptions if it's not?